lite/xeus-environment.yml

```
@@ -3,7 +3,7 @@ channels:
   - https://repo.mamba.pm/emscripten-forge
```

@agriyakhetarpal do you know why we were using https://repo.mamba.pm/emscripten-forge rather than https://prefix.dev/channels/emscripten-forge-dev? The former was last updated on 2025-07-03 and has a very old xeus-r (`0.5.0`)

```suggestion
  - https://repo.prefix.dev/emscripten-forge-dev
```